### ISSUE TITLE
Non-visible fields in admin interface are still tabbable and read out when sidebar closed

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov#565]
### Problem/Motivation
When you are editing content on devices 64em or larger and hide the sidebar, the content in the sidebar can still be tabbed to. This is an accessibility issue as these fields are read out to assistive technology users, even though that content is not intended to be interacted with. Additionally, this is an inconvenience to any users using tab to navigate.

Fails WCAG 2.1 - 1.3.2 & 2.4.3 Https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html and https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html

### Steps to reproduce
1. Open a browser window larger than 64em
2. Create new content
3. Hide the sidebar
4. Tab through the fields in the content editing view
Observe that you must tab several times before reaching new buttons.

If using a screenreader, observe something like this being read out as you tab through:

```
Revision log message edit multi line Briefly describe the changes you have made. blank
Save as: combo box Draft collapsed
Review dateNot reviewed yet toggle button pressed expanded
Content reviewed check box not checked I have reviewed this content.
Menu settingsNot in menu toggle button not pressed collapsed
```

### Proposed resolution
CSS change to set the content of the sidebar as visibility: hidden or similar when the sidebar is closed.

**Comment by keelanfh** [comment]
Upstream issue (with patch, currently ~RTBC~ merged and awaiting a Gin release): https://www.drupal.org/project/gin/issues/3366228

**Comment by keelanfh** [comment]
Closing as fixed in new release of Gin - https://www.drupal.org/project/gin/releases/8.x-3.0-rc4
